tests/integration/test_vulnerability_detector/test_feeds/debian/test_invalid_values_debian_feed.py

```diff
@@ -0,0 +1,154 @@
+
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import itertools
+
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools import file
+from wazuh_testing.tools.utils import replace_regex_group
+from wazuh_testing.tools.services import control_service
+import wazuh_testing.vulnerability_detector as vd
```

Comment:
```suggestion
import os
import pytest
import itertools
import wazuh_testing.vulnerability_detector as vd

from wazuh_testing.tools import file, LOG_FILE_PATH
from wazuh_testing.tools.monitoring import FileMonitor
from wazuh_testing.tools.configuration import load_wazuh_configurations
from wazuh_testing.tools.utils import replace_regex_group
from wazuh_testing.tools.services import control_service
```

Reply:
Done in ba7bcc0729d9649d5d646f0d5ea80a440267f65b